UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Services/CallbackService.cs

```diff
@@ -42,5 +54,56 @@ public static bool SaveCallbackRequest(CallbackRequest callbackRequest, string s
             }
 
         }
+
+        public bool SaveCommandCallbackRequest(string subscriptionId, HttpStatusCode? httpStatusCode)
+        {
+            try
+            {
+                CommandCallbackRequest? commandCallbackRequest = s_commandCallbackRequestList.FirstOrDefault(a => a.SubscriptionId == subscriptionId);
+
+                if (commandCallbackRequest != null)
+                {
+                    if (httpStatusCode == null)
+                    {
+                        s_commandCallbackRequestList.Remove(commandCallbackRequest);
+                    }
+                    else
+                    {
+                        commandCallbackRequest.HttpStatusCode = httpStatusCode ?? _noContent;
```

Comment:
Change this as per above change:
```suggestion
                        commandCallbackRequest.HttpStatusCode = httpStatusCode ?? NoContent;
```